### ISSUE TITLE
Files created inside VS Code but saved outside of current project are not found in Recently Opened

### ISSUE THREAD

**Issue by bakutheleo** [microsoft/vscode#108952]
Issue Type: <b>Bug</b>

I created a new file, added some text, then saved it as a new name to my Documents folder.  I then closed it. After that I went to reopen it in the same Visual Studio Code session (I had not restarted VSC) and the file was not listed in my recently opened files. I tried opening a new VSC session (without closing my current one) and it was not listed there either. I also tried shuttind down VSC completely then reopening and it is still not listed. I have seen this (or similar) behavior at least once before.

VS Code version: Code 1.50.0 (93c2f0fbf16c5a4b10e4d5f89737d9c2c25488a3, 2020-10-07T06:10:52.432Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz (8 x 2712)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.86GB (21.78GB free)|
|Process Argv|--crash-reporter-id f33085d1-fe45-4a14-9ef7-725c2f833c89|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (2)</summary>

Extension|Author (truncated)|Version
---|---|---
remote-wsl|ms-|0.50.1
powershell|ms-|2020.6.0

</details>

**Comment by jsunsawyer** [comment]
This may be related, but I just ran into an issue where after I opened a project via terminal, 2 files from another open workspace were showing up in my newly opened workspace.

**Comment by bpasero** [comment]
Recently opened are only filled from actions such as File > Open. Otherwise use the recently editor list in Cmd+P.

**Comment by bakutheleo** [comment]
Title of this report was changed to an incorrect description. File was not saved outside current project. This will reproduce the behavior:

1. Create a new file in VSC (VS Code)
2. Type some text
3. File->Save As within VSC and save the file somewhere
4. Close the file in VSC
5. File->Open Recent in VSC and file is not listed

Are you saying that Save As will not update recently opened files? I would expect it to do so; that is normal behavior on every other application I have seen (For example, Word). None of the steps I described involve opening or saving the file outside VS Code.
